src/LoginPage/components/RegisterModal.js

```diff
@@ -32,7 +34,7 @@ class RegisterModal extends Component {
         <div>
           {" "}
         </div>
-        <button id="navbar-button" label="Scrollable Dialog" onClick={this.handleOpen} > Register </button>
+        <Link id="registerBtn" to='#' label="Scrollable Dialog" onClick={this.handleOpen}><h2>create new account</h2></Link>
```

Comment:
I think capitalizing every word is the better approach here, but I'd like to see this in action again.

Reply:
i think everyone agreed that it looks better in all lower case.